app_dart/lib/src/request_handlers/refresh_chromebot_status.dart

```diff
@@ -119,14 +119,17 @@ class RefreshChromebotStatus extends ApiRequestHandler<Body> {
         final Task update = datastoreTask.task;
         update.status = latestLuciTask.status;
 
+        final CiYaml ciYaml = await scheduler.getCiYaml(datastoreTask.commit);
```

Comment:
`scheduler.getCiYaml` is cached making it trivial to pull up. Since this logic is only a for loop against existing tasks, is it ok to leave as is? It's difficult to refactor this code to be a double for loop of commit to tasks